corehq/apps/registration/tests/test_utils.py

```diff
@@ -105,7 +107,7 @@ def test_domain_is_not_active_for_new_user(self):
         self.assertFalse(domain.is_active)
 
     @suspend(domain_tombstone_patch)
-    @mock.patch.object(BillingContactInfo.objects, 'get_or_create', _issue_initializing_domain)
+    @mock.patch.object(type(BillingContactInfo.objects), 'get_or_create', _issue_initializing_domain)
```

Comment:
This doesn't necessarily feel like an improvement as I think the previous version is easier to understand. Is this to make `_issue_initializing_domain` more flexible with other types? And is that necessary if we are only using it here anyway?

Reply:
I agree with this comment, I don't quite understand how this commit improves things.

Reply:
I agree, but it is unfortunately necessary to get tests to pass on Github. (The previous patch worked when I ran tests locally.) I was not able to determine the precise cause, but something in some other test is messing up the model state such that `BillingContactInfo.objects` is returning a new instance of `Manager` in the test, so the patch as done previously had no effect.

The "Improve domain initialization patch" commit message could have been phrased better.

Reply:
Ah, got it, that's unfortunate but it makes sense.